restli-server/src/main/java/com/linkedin/restli/server/ResourceContext.java

```diff
@@ -236,6 +236,10 @@
    * 2. the server is configured to return default value for every request-response pair
    * @return true if the service need to fill in default value when the field of the return value record is absent
    */
-  boolean isFillInDefaultsRequested();
-  void setFillInDefaultValues(boolean fillInDefaultValues);
+  default boolean isFillInDefaultsRequested()
+  {
+    return false;
+  }
+
+  default void setFillInDefaultValues(boolean fillInDefaultValues) {}
```

Comment:
why is this no-op function a default method?

Reply:
This pair of set/get appeared to cause learning-api MP's not override interface method issue.
```
frontend/test/com/linkedin/learning/api/enterprise/mock/MockResourceContext.java:21: error: MockResourceContext is not abstract and does not override abstract method setFillInDefaultValues(boolean) in ResourceContext
public class MockResourceContext implements ResourceContext {
```
So I have this to represent any other similar situation

Reply:
But before you made this change, the original method is also not default, how did it work previously for learning-api?

Reply:
Something probably changed in deco, this is the wc-test I did for default serialization on Monday, https://crt.prod.linkedin.com/#/testing/executions/3150a0af-7b96-44e5-8f9a-3e4e6d352c53/child-executions
and I can't find learning-api in child execution, so I think it might be new

Reply:
I will speak with offline about it

Reply:
I think this one is unrelated to "Add pdsc support for extension schema resolver", right?

Reply:
yes